### PR TITLE
Task 1012

### PR DESCRIPTION
**Pull request by tcavaletto**

### CODE REVIEW

File: app/controllers/search_controller.rb [hunk 1]
```diff
@@ -34,4 +34,28 @@ def uc_search
     @curricula = @curr_search.results
     @number_of_curriculum_matches = @curr_search.total
   end
+
+  def s_follow
+    @user = User.find_by_username(params[:username])
+    if current_user
+      Watching.create(user_id: current_user.id, peer_id: @user.id)
+      flash[:notice] = "You are now following #{@user.name}."
```
Comment (bammons) [inline app/controllers/search_controller.rb]: Take a look in app/helpers/application_helpers.rb

We made custom flash messages. We also defined notice and error, so if you knew about it already and this is how you made it I think you should use flash[:success] here instead

File: app/controllers/search_controller.rb [hunk 2]
```diff
@@ -34,4 +34,28 @@ def uc_search
...
+
+  def s_unfollow
+    @user = User.find_by_username(params[:username])
+    if current_user
+      Watching.where('user_id=? AND peer_id=?', current_user.id, @user.id).destroy_all
+      flash[:notice] = "You are no longer following #{@user.name}."
+    else
+      flash[:error] = "You must login to unfollow #{@user.name}.".html_safe
+    end
+    @search_path = '/search/uc_search/?utf8=%E2%9C%93&query=' + params[:query]
```
Comment (bammons) [inline app/controllers/search_controller.rb]: I don't think you should manually put in the query for the url.

Maybe try this: <a href="http://stackoverflow.com/questions/5433591/rails-redirect-with-params">here</a>

If it doesn't work let me know

**Comment by tcavaletto** [comment]
Fixed.

**Comment by bammons** [comment]
awesome job +1

**Comment by Linkster700** [comment]
+1 nice job on the flash messages

**Pull request merged**